index.js

```diff
@@ -0,0 +1,2 @@
+let nombreUsuario = prompt(`Ingrese su nombre`);
+alert(`Hola, ${nombreUsuario}`);
```

Comment:
no usar interpolacion
```suggestion
alert('Hola,' + nombreUsuario);
```